datahub-web-react/src/app/ingestV2/source/IngestionSourceList.tsx

```diff
@@ -80,8 +109,13 @@ const removeExecutionsFromIngestionSource = (source) => {
     return undefined;
 };
 
-export const IngestionSourceList = () => {
-    const entityRegistry = useEntityRegistry();
+interface Props {
+    showCreateModal?: boolean;
+    setShowCreateModal?: (show: boolean) => void;
+}
+
+export const IngestionSourceList = ({ showCreateModal, setShowCreateModal }: Props) => {
```

Comment:
i thought we were going to have these changes behind a feature flag? is that not the case?